azure.yaml

```diff
@@ -29,6 +29,21 @@
   date: 2019-10-25T10:00:00Z
   version: 9.0.0
 - active: true
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: azure-operator
+      version: 2.6.2
+    - name: cert-operator
+      version: 0.1.0
+    - name: chart-operator
+      version: 0.7.0
+    - name: cluster-operator
+      provider: azure
+      version: 0.20.0
+  date: 2019-12-31T10:00:00Z
+  version: 8.6.0
+- active: false
```

Comment:
This should be true, right?

Reply:
Wait no - I am dumb ignore me.

Reply:
It is version 8.4.0. Do we want it active as 9.0.0 and 8.6.0 are active?

Reply:
It happens :)